requirements/pytorch/base.txt

```diff
@@ -10,3 +10,4 @@ torchmetrics>=0.7.0, <0.10.1  # needed for using fixed compare_version
 packaging>=17.1, <=23.0
 typing-extensions>=4.0.0, <=4.4.0
 lightning-utilities>=0.6.0.post0, <0.7.0
+onnx<1.14.0
```

Comment:
Why not keep this in the test requirements?

Reply:
becase it is needed for PT 2.0, they have it as optional package so it crash with error you need to install it :)

Reply:
and in tets we still keep runitme version... :chipmunk:

Reply:
I don't think that's right. The onnx installation should still be optional: https://github.com/Lightning-AI/lightning/pull/16891

Reply:
ok, lets merge #16764 here instead of master :otter: